packages/webdriver-image-comparison/src/helpers/utils.ts

```diff
@@ -24,7 +24,9 @@ export function getAndCreatePath(folder: string, options: GetAndCreatePathOption
     const subFolder = savePerInstance ? instanceName : ''
     const folderName = join(folder, subFolder)
 
-    ensureDirSync(folderName)
+    if (!existsSync(folderName)) {
+        mkdirSync(folderName, { recursive: true })
```

Comment:
**Non-blocking**: We may not need `!existsSync` because `mkdirSync` does not throw when the directory already with `recursive: true`.

Reply:
Thanks!